moveit_kinematics/CMakeLists.txt

```diff
@@ -6,18 +6,19 @@ find_package(moveit_common REQUIRED)
 moveit_package()
 
 find_package(ament_cmake REQUIRED)
+
 find_package(Eigen3 REQUIRED)
```

Comment:
Thanks for the PR!

I have some questions that @ChrisThrasher would probably know the answer to:

- Boost is only under `set(THIS_PACKAGE_INCLUDE_DEPENDS
  Boost`
- Should Eigen3 go under 
`set(THIS_PACKAGE_INCLUDE_DEPENDS` as well?

Reply:
I believe so if Eigen types appear in the interface of this library.

Reply:
Well, Boost isn't part of the interface of this library. So, remove Boost from `set(THIS_PACKAGE_INCLUDE_DEPENDS`. Make sense, @ChrisThrasher?

Eigen is part of the interface so I think it should be added to `set(THIS_PACKAGE_INCLUDE_DEPENDS`

Reply:
Yes, Eigen3 should be set as a dependency. If we are exporting headers with Eigen includes, we should probably even use `eigen3_cmake_module` for better ament support https://github.com/ros2/eigen3_cmake_module.

Reply:
That seems right to not export Boost. No need to export private dependencies since downstream projects ought not know about them.